spring-kafka-3/src/main/java/com/baeldung/spring/kafka/start/stop/consumer/KafkaListenerControlService.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.spring.kafka.start.stop.consumer;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.config.KafkaListenerEndpointRegistry;
+import org.springframework.kafka.listener.MessageListenerContainer;
+import org.springframework.stereotype.Service;
+
+@Service
+public class KafkaListenerControlService {
+
+    @Autowired
+    private KafkaListenerEndpointRegistry registry;
+
+    // Method to start a listener
```

Comment:
We can remove comments // Method to start a listener and // Method to stop a listener

Reply:
Removed comments